CoC.txt

```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+
+- Assume good faith and good intentions. Disagreements — both social and
+  technical — happen all the time. Our diverse backgrounds give us the ability
+  to see things from many different perspectives, but they can also lead to
+  misunderstandings. It is important that we resolve disagreements
+  constructively, and that we do not jump to conclusions about each other's
+  motives.
+
+- Be welcoming, considerate, and respectful. We strive to be a community that
+  welcomes and supports people of all backgrounds and identities. This includes,
+  but is not limited to, members of any race, ethnicity, culture, national
+  origin, color, immigration status, social and economic class, educational
+  level, sex, sexual orientation, gender identity and expression, age, size,
+  family status, political belief, religion and mental and physical ability.
+
+- Be careful in the words you choose. We are a community of professionals, and
+  we conduct ourselves professionally. Exclusionary behavior is absolutely
+  unacceptable. This includes, but is not limited to:
+
+  - Discriminatory (e.g., racist or sexist) jokes and language
+  - Inappropriate names for user accounts, servers, files, repositories, etc.
+  - Posting sexually explicit or violent material
+  - Personal insults
+  - Unwelcome sexual attention
+  - Advocating for, or encouraging, any of the above behavior
+
+- Listen and communicate openly and honestly. Yield the floor to
+  others, and help to make sure that everyone gets heard. In other words, try to
+  be your best self. In doing so, you contribute the health and longevity of
+  this community.
+
+As a member of the SecureDrop community, we ask you to never engage in the
+following behaviors:
+
+- Intimidation, harassments, or insults.  This includes but is not limited to:
+  - Physical intimidation or threats against someone's physical safety
+  - Obscene or intimidating gestures
+  - Stalking
+  - Demeaning another person
+  - Unwelcome following
+  - Enlisting the help of others, whether in person or online, in order to
+    target an individual
+  - Taking photographs, video, or audio recordings or recordings without consent
+  - Shouting
+  - Sustained disruption of talks and events
+  - Disruption of meetings
+  - Inappropriate physical contact
+  - Unwelcome sexual attention
+  - Posting (or threatening to post) other people's personally identifying
+    information ("doxing")
+
+- Following the letter of this Code of Conduct while disregarding its spirit.
+  When judging whether certain behavior represents a violation of this code, we
+  will consider a violation in spirit (e.g., clearly behaving in a damaging or
+  obnoxious manner in a way not explicitly specified) to be no different from
+  any other violation of this code. That includes trolling and other forms of
+  consistently disruptive behavior.
+
 ================================================================================
 License and Attribution
```

Comment:
Just since I'm trying to keep up with licensing / attribution, if you took any of this directly or nearly, could you make sure to include that down here. I'd like to give credit where it's due. :)

Reply:
Thanks for flagging, added FPF and OpenStack as inspirations, as well.